docs/pages/reference/cli.mdx

```diff
@@ -327,6 +327,106 @@ $ tsh play --format=pty 1fe153d1-ce8b-4ef4-9908-6539457ba4ad.tar
 $ tsh play --format=json ~/play/0c0b81ed-91a9-4a2a-8d7c-7495891a6ca0.tar | jq '.event
 ```
 
+### tsh proxy db
+
+Start local TLS proxy for database connections when using Teleport in single-port mode.
+Database connections can be made with certificates through database clients or tunnel mode
```

Comment:
I think we should use the active voice here—instead of saying "can be made", we should clarify what is making the connections.